src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/runtime.Linux.Microsoft.NETCore.Runtime.CoreCLR.props

```diff
@@ -6,7 +6,7 @@
   </PropertyGroup>
   <ItemGroup>
     <NativeBinary Include="$(BinDir)libcoreclr.so" />
-    <NativeBinary Condition="'$(_PlatformDoesNotSupportNiFiles)' != 'true'" Include="$(BinDir)libcoreclrtraceptprovider.so" />
+    <NativeBinary Condition="'$(Platform)' != 'armel'" Include="$(BinDir)libcoreclrtraceptprovider.so" />
```

Comment:
It looks strange to conditionalize it here based on the platform when you really mean Tizen. People can do armel builds for other platforms where the trace provider can be built, right?
Actually, the previous condition was also strange.
@gkhanna79, can you see a better way to conditionalize the inclusion of the libcoreclrtraceptprovider.so in the package?